src/app/integrations/list/list.component.html

```diff
@@ -106,27 +106,28 @@ <h4 class="modal-title pull-left">Confirm {{ getActionTitle() }}</h4>
               </h3>
             </div>
 
-            <div *ngIf="integration.currentStatus !== 'Pending'">
-              <span *ngIf="integration.currentStatus === 'Activated'"
-                    class="label label-primary"
-                    (click)="requestDeactivate(integration)">Active</span>
-              <span *ngIf="integration.currentStatus === 'Deactivated'"
-                    class="label label-default"
-                    (click)="requestActivate(integration)">Inactive</span>
-              <span *ngIf="integration.currentStatus === 'Draft'"
-                    class="label label-warning"
-                    disabled>Draft</span>
-              <span *ngIf="integration.currentStatus === 'Deleted'"
-                    class="label label-danger"
-                    disabled>Deleted</span>
+            <div *ngIf="integration.currentStatus !== 'Pending' && integration.currentStatus !== 'Draft'">
```

Comment:
maybe just check if it's `===` `'Activated'` or `'Deactivated'` instead?  Otherwise this might show up in the `Deleted` case?